<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/img/icon/main.png</normaloff>:/images/img/icon/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background:rgba(0,0,0,0.4);
	border-radius:15px;
}

QFrame:hover{
	background:rgba(0,0,0,0.6);
}

QLineEdit{
	font-size:25px;;
	color:white;
	border:none;
	background:transparent;
	border-bottom:1px solid white;
}

QLineEdit:hover{
	background:rgba(0,0,0,0.04);
}


QPushButton{
	background:rgb(83, 152, 255);
	border-radius:10px;
	color:#fff;
}

QPushButton:hover{
	border:none;
	background:rgb(68, 117, 222)
}

QCheckBox{
	color:silver;
	background:transparent
}
QCheckBox:hover{
	color:white;
}</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background:rgba(0,0,0,0.60);
}
QFrame{

border-radius:none;

}

QFrame:hover{
background:rgba(0,0,0,0.75);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>40</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
color:silver;

border-radius:none;
}


QPushButton:hover{
background:red;
color:white
}</string>
    </property>
    <property name="text">
     <string>âœ•</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>40</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
border-radius:none;
}


QPushButton:hover{
background:rgba(0, 0, 0,0.6)
}</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:silver;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Ro'yxatdan o'tish</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>621</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	border-bottom-left-radius: 25px;
	border-bottom-right-radius: 25px;
	background-image: url(:/images/img/background/reg_1.jpg);
}</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>560</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="name_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>131</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="placeholderText">
     <string>Ism</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="baslik_3">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>10</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:40px;
color:white;
background:transparent;</string>
    </property>
    <property name="text">
     <string>RESTORAN NOMI</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="baslik_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>80</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:18px;
color:white;
background:transparent;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Buyurtma berish tizimi</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Familiya</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Parol takror</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Parol</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>+\9\9\8(99)-999-99-99</string>
    </property>
    <property name="text">
     <string>+998()---</string>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Telefon raqam</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_createAccount_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>400</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Ro'yxatdan o'tish</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
 </widget>
 <tabstops>
  <tabstop>name_2</tabstop>
  <tabstop>surname_2</tabstop>
  <tabstop>phone_2</tabstop>
  <tabstop>password_2</tabstop>
  <tabstop>password_3</tabstop>
  <tabstop>btn_createAccount_2</tabstop>
  <tabstop>btn_close</tabstop>
  <tabstop>btn_minimize</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
